src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,20 @@
+package duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Scanner;
+
+public class Duke {
+    public static void main(String[] args) throws FileNotFoundException {
+        //File saveFile = FileHandler.loadOrCreateSaveFile("data/duke.txt");
```

Comment:
Consider removing [dead code](https://nus-cs2113-ay2122s1.github.io/website/se-book-adapted/chapters/codeQuality.html#delete-dead-code) like this to increase code readability.